src/components/filter/Filter.jsx

```diff
@@ -1,78 +1,247 @@
+import { useProduct } from "../../context/product-context";
```

Comment:
Since there will be multiple contexts made for this project in the future, I would suggest you to export them from a single index.js file that you can create inside the context folder and import your context from that file over here.

Reply:
I was planning to add index.js after I add more contexts.  I will do that. Thanks.